src/posts/2020-10-15-the-burden-of-choice/index.md

```diff
@@ -1,98 +1,167 @@
 ---
 path: /posts/the-burden-of-choice/
-title: "The Burden of Choice"
+title: The Burden of Choice
 author: martinho-aragao
 date: 2020-10-15
-seoImage: ./seo.png
+seoImage: ./seo.jpeg
 tags:
   - product
-intro: "Whoever would be able to crack the prioritisation code that leads products towards success could potentially spearhead most economic developments. What is this burden of choice in product development?"
+intro: >
+      Whoever would be able to crack the prioritisation code that leads products towards success could potentially spearhead most economic developments. 
+      What is this burden of choice in product development?
```

Comment:
What's up with the indentation here? It's only supposed to be 2 spaces.

```suggestion
  Whoever would be able to crack the prioritisation code that leads products towards success could potentially spearhead most economic developments.
  What is this burden of choice in product development?
```

Reply:
Ahahah

Haven't re-configured my editors properly. It's a tab 😱